user_docs/en/userGuide.t2t

```diff
@@ -1617,7 +1619,7 @@ You can configure reporting of:
 To toggle these settings from anywhere, please assign custom gestures using the [Input Gestures dialog #InputGestures].
 
 ==== Report formatting changes after the cursor ====[DocumentFormattingDetectFormatAfterCursor]
-If enabled, this setting tells NVDA to try and detect all the formatting changes on a line as it reports it, even if doing this may slow down NVDA's performance.
+If enabled, this setting tells NVDA to try and detect all the formatting changes on a line as it reports it - even if doing this may slow down NVDA's performance.

```

Comment:
This seems a little bit strange to me, I would expect a comma here or perhaps a semi-colon.

Also for the change in restoring the default settings below this.